modules/docs/Makefile

```diff
@@ -6,8 +6,8 @@ docs/deps::
 ## Update `docs/targets.md` from `make help`
 docs/targets.md: docs/deps
 	@( \
-		echo "## Makefile Targets"; \
-		echo '```'; \
+		echo "# Makefile Targets"; \
```

Comment:
This is erroneous. The file is included in the `README.md` and should be at the `##` depth. Can we configure the linter to ignore this file? Many linters allow adding an exclusion. 

I guess we could add this file also to `.gitignore` and not check it in.

https://github.com/igorshubovych/markdownlint-cli#ignoring-files

Reply:
Good catch. I didn't notice it's included in README. 
Yep, I'll use the ignore file to mitigate that.

Reply:
⬆️ What you provided is actually different repository.
This one is markdownlint used by superlinter
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041